contracts/stork_pyth_adapter/contracts/StorkPythAdapter.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.8.24 <0.9.0;
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
+
+/**
+ * @title A port of the IPyth interface that supports Stork price feeds
+ */
+contract StorkPythAdapter is IPyth {
+    IStork public stork;
+    int32 private exponent = -18;
```

Comment:
Pyth's contract returns `-8` currently since you need to multiply their price integer by `10^-8` to convert it to a human readable price. We use -18 instead.